sensor/NmapSensor.js

```diff
@@ -213,6 +213,10 @@ class NmapSensor extends Sensor {
     return Promise.all(this.networkRanges.map((range) => {
 
       log.info("Scanning network", range, "to detect new devices...");
+      if (range.endsWith('/8')) {
```

Comment:
shall we force to scan /24 when /8 is specified?

Reply:
Make sure there is a visual alarm, and then force /24, for the blue in the future we may expend more.